docker/Dockerfile.ROS

```diff
@@ -50,7 +50,7 @@ RUN curl -sSL https://raw.githubusercontent.com/ros/rosdistro/master/ros.key -o
 RUN apt-get update && apt-get install -y ros-dev-tools
 
 # Setup workspace directory
-RUN mkdir -p /workspace
+RUN mkdir -p /workspace && chown -R $USERNAME:$USERNAME /workspace
```

Comment:
why is this needed?? could you add some reference here to some docs where we can learn more about this?